miden/tests/integration/stdlib/crypto/fri/mod.rs

```diff
@@ -15,6 +15,7 @@ pub use verifier_fri_e2f4::*;
 mod remainder;
 
 #[test]
+#[ignore = "the unbound integrated test struct needs to be refactored to use merkle store"]
```

Comment:
We should fix this (perhaps in a different PR) before we merge this PR.

Reply:
Added as `Must have` as it might reveal a bug either in the advice provider or the merkle store backend